cli/api_cmds.go

```diff
@@ -379,15 +379,15 @@ as the JSON representation of the following struct.
 
 The Def and Example structs are defined as follows in the Sourcegraph API.
 
-[[.code "src/api_cmds.go" "APIDescribeCmdOutputQuickHack"]]
+[[.code "cli/api_cmds.go" "APIDescribeCmdOutputQuickHack"]]
 
 [[.code "https://raw.githubusercontent.com/sourcegraph/go-sourcegraph/6937daba84bf2d0f919191fd74e5193171b4f5d5/sourcegraph/defs.go" 105 113]]
```

Comment:
Yes. The current data structures would be in `https://sourcegraph.com/github.com/sourcegraph/go-sourcegraph@nodb/.tree/sourcegraph/sourcegraph.pb.go`